README.md

```diff
@@ -8,7 +8,6 @@
   - [Supported SQL Server Versions](#supported-sql-server-versions)
   - [Known Issues](#known-issues)
     - [Output Bindings](#output-bindings)
-    - [Trigger Bindings](#trigger-bindings)
```

Comment:
My assumption is that links from docs will need to be updated. Is this correct?

@dzsquared as FYI